bindings/c/include/kllvm-c/kllvm-c.h

```diff
@@ -0,0 +1,68 @@
+#ifndef KLLVM_C_H
+#define KLLVM_C_H
+
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Opaque types */
+
+typedef struct kore_pattern kore_pattern;
+typedef struct kore_sort kore_sort;
+typedef struct kore_symbol kore_symbol;
+typedef struct block block;
+
+/* KOREPattern */
+
+char *kore_pattern_dump(kore_pattern const *);
+
+void kore_pattern_free(kore_pattern const *);
+
+kore_pattern *kore_pattern_new_token(char const *, kore_sort const *);
+kore_pattern *kore_pattern_new_injection(
+    kore_pattern *, kore_sort const *, kore_sort const *);
+kore_pattern *kore_pattern_make_interpreter_input(kore_pattern *);
+
+kore_pattern *kore_composite_pattern_new(char const *);
+kore_pattern *kore_composite_pattern_from_symbol(kore_symbol *);
+void kore_composite_pattern_add_argument(kore_pattern *, kore_pattern *);
+
+kore_pattern *kore_string_pattern_new(char const *);
+
+block *kore_pattern_construct(kore_pattern const *);
+char *kore_block_dump(block *);
+
+// Expects the argument term to be of the form:
+//   sym{}(BOOL)
```

Comment:
can we change these to /* */ style comments so my preprocessor doesn't complain?